extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -127,8 +129,8 @@ core::Property ConsumeWindowsEventLog::EventHeaderDelimiter(
 core::Property ConsumeWindowsEventLog::EventHeader(
   core::PropertyBuilder::createProperty("Event Header")->
   isRequired(false)->
-  withDefaultValue("LOG_NAME=Log Name, SOURCE = Source, TIME_CREATED = Date,EVENT_RECORDID=Record ID,EVENTID = Event ID,TASK_CATEGORY = Task Category,LEVEL = Level,KEYWORDS = Keywords,USER = User,COMPUTER = Computer, EVENT_TYPE = EventType")->
-  withDescription("Comma seperated list of key/value pairs with the following keys LOG_NAME, SOURCE, TIME_CREATED,EVENT_RECORDID,EVENTID,TASK_CATEGORY,LEVEL,KEYWORDS,USER,COMPUTER, and EVENT_TYPE. Eliminating fields will remove them from the header.")->
+  withDefaultValue("LOG_NAME=Log Name, SOURCE = Source, TIME_CREATED = Date,EVENT_RECORDID=Record ID,EVENTID = Event ID,TASK_CATEGORY = Task Category,LEVEL = Level,KEYWORDS = Keywords,USER = User,COMPUTER = Computer, EVENT_TYPE = EventType")->  // NOLINT linelength
+  withDescription("Comma seperated list of key/value pairs with the following keys LOG_NAME, SOURCE, TIME_CREATED,EVENT_RECORDID,EVENTID,TASK_CATEGORY,LEVEL,KEYWORDS,USER,COMPUTER, and EVENT_TYPE. Eliminating fields will remove them from the header.")-> // NOLINT linelength
```

Comment:
Could we rather split it in multiple lines?

Reply:
The reason I suggested the split is that we already don't follow the google guideline in this regard, as the current maximum line length is set to 200 characters which is quite permissive. Also we already have split the long descriptions in other processors as well. But as it is an exception to the rule I'm not against the idea of keeping it in a single line, it also has the advantage of being able to search for the same description in both the codebase and the *.md files.

I also support moving the `->` operators to the beginning of the lies.

Reply:
Splitting them like this seems like a common practice in this extension.
https://github.com/apache/nifi-minifi-cpp/blob/main/extensions/windows-event-log/ConsumeWindowsEventLog.cpp#L81-L95
So at least now we would be consistent with the style used here.

Reply:
I agree the -> operator is more readable in the beginning of the lines, however I think it would be better to change them all at once in a separate PR (maybe with some clever regex?) because this is the style used in all files in the wel extension

I think we could also document these rules (with the differences which we use apart from the google style guide), and enforce them afterwards.

Reply:
If you prefer to split, then don't feel blocked by me.